model/validate.go

```diff
@@ -40,7 +40,7 @@ func ValidateTask(taskId string, checkSecret bool, r *http.Request) (*task.Task,
 	if checkSecret {
 		secret := r.Header.Get(evergreen.TaskSecretHeader)
 		if secret != t.Secret {
-			return nil, http.StatusConflict, errors.Errorf("Wrong secret sent for task '%s'", taskId)
+			return nil, http.StatusConflict, errors.Errorf("wrong secret sent for task '%s'", taskId)
```

Comment:
`invalid secret sent...`

Reply:
Missed this.

Reply:
I think this one actually makes sense since it's comparing the sent secret string against the actual one.